client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.html

```diff
@@ -7,7 +7,10 @@ <h3 class="oui-header_3" data-translate="cpc_loadbalancer_discover_title"></h3>
             <img data-ng-src="assets/images/cloud/loadbalancer/loadbalancer-schema.png" alt="{{cpc_loadbalancer_discover_title | translate}}"/>
         </div>
         <div class="discover-iplb__button">
-            <a class="oui-button oui-button_primary" data-ng-href="{{CloudProjectComputeLoadbalancerCtrl.urls.website_order.load_balancer[CloudProjectComputeLoadbalancerCtrl.locale]}}" data-translate="cpc_loadbalancer_order"></a>
+            <oui-button data-variant="secondary"
```

Comment:
This should be a link instead of a button :)

Reply:
I dont know if at-internet works with `<a>` tags. Ill check